### ISSUE TITLE
What settings to use for gapless playback?

### ISSUE THREAD

**Issue by jbrepogmailcom** [arnesongit/plugin.audio.tidal2#9]
Hello I have gaps in my FLAC playback by Tidal2 on RPi4. I think I could have had gapless before. The comments in issue #5 do not help. Are there any specific setting for gapless to work?

**Comment by arnesongit** [comment]
Please try to deactivate the setting "Use inputstream.ffmpegdirect as default player".

**Comment by jbrepogmailcom** [comment]
Thank you very much, but I already had this setting off. I tried various combinations of 2nd, 3rd and 4th setting (for FLAC) and nothing helped. So for now I am staying with gaps and this (I have KODI 18 on RPi4):
2022-12-13 19:27:54.835 T:2109887296  NOTICE: [TIDAL2] Got stream with soundQuality:LOSSLESS, codec:FLAC
2022-12-13 19:27:54.841 T:2109887296  NOTICE: [TIDAL2] Got Dash stream with MimeType: audio/x-flac
2022-12-13 19:27:54.907 T:2109887296  NOTICE: [TIDAL2] Using inputstream.ffmpegdirect for HLS playback

I also see othe message below:
NOTICE: Creating audio stream (codec id: 86028, channels: 2, sample rate: 44100, no pass-through)
I would like to have pass-through but I guess that is not your concern, it is problem of KODI 

I found advice to use DLNA player on Android that supports Tidal, so I use that with KODI as DLNA client. Works well, gapless.
